cognite/client/_api/data_modeling/instances.py

```diff
@@ -92,6 +94,8 @@
 
 logger = logging.getLogger(__name__)
 
+Source: TypeAlias = Union[SourceDef, View, ViewIdentifier]
```

Comment:
I think I was being ambiguous in my other comment. IMO this should accept a SourceSelector instead of a SourceDef. Do we really need SourceDef at all?